CK3toEU4/Source/CK3World/Titles/LandedTitles.cpp

```diff
@@ -116,15 +116,15 @@ void CK3::LandedTitles::linkTitles(const Titles& titles)
 	{
 		if (!landedTitle.second->getCapital())
```

Comment:
This function calls getCapital() three times. It's probably better to call it once and cache the result.